src/main/groovy/com/jakewharton/sdkmanager/internal/PackageResolver.groovy

```diff
@@ -134,6 +142,11 @@ class PackageResolver {
   }
 
   def resolveSupportLibraryRepository() {
+
+    project.repositories.maven {
+      url = androidRepositoryDir
```

Comment:
I have moved adding android repository url above dependency scan routine.
This is necessary because now we are not just parsing strings at the `build.gradle` file, but gradle actually tries to resolve dependencies. And to be able to resolve the "up-to-date" case gradle needs m2repository link to resolve support library if it's repository is present.